types.d.ts

```diff
@@ -1,3 +1,7 @@
 import { Source } from 'callbag'
 
-export default function fromEvent(node: EventTarget, name: string): Source<Event>;
+export default function fromEvent<T extends Event = Event>(
```

Comment:
please review this change extra carefully - im not a TS expert, beside adjusting the types to account for new argument I've also made the whole thing generic, which makes sense, right? MAYBE this even could be improved further by using conditional types, but this is over my head right now

Reply:
Looks safe. The best would be to implement in TypeScript, not JavaScript, but I believe this will work